adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/CoverageRDD.scala

```diff
@@ -162,13 +188,29 @@ case class DatasetBoundCoverageRDD private[rdd] (
 
   def toFeatures(): FeatureRDD = {
     import dataset.sqlContext.implicits._
-    DatasetBoundFeatureRDD(dataset.map(_.toSqlFeature), sequences)
+    DatasetBoundFeatureRDD(dataset.map(_.toSqlFeature), sequences, samples)
   }
 
+  /**
+   * Replaces the sequence dictionary attached to a DatasetBoundCoverageRDD.
+   *
+   * @param newSequences The new sequence dictionary to attach.
+   * @return Returns a new DatasetBoundCoverageRDD with the sequences replaced.
+   */
   def replaceSequences(
     newSequences: SequenceDictionary): CoverageRDD = {
     copy(sequences = newSequences)
   }
+
+  /**
+   * Replaces the sample metadata attached to the DatasetBoundCoverageRDD.
+   *
+   * @param newSamples The new sample metadata to attach.
+   * @return A DatasetBoundCoverageRDD with new sample metadata.
+   */
+  def replaceSamples(newSamples: Iterable[Sample]): CoverageRDD = {
```

Comment:
This method is defined with scaladoc in `MultisampleGenomicDataset`, could we use `override def` instead?